conf/smRNA.config

```diff
@@ -1,143 +1,63 @@
 
 /*
  * -------------------------------------------------
- *  Nextflow config file for small RNA reference paths
+ * Nextflow config file for small RNA reference paths
```

Comment:
Can we rename this file to `smrna.config`?

Reply:
Done: [a1eca9e](https://github.com/nf-core/clipseq/pull/28/commits/a1eca9e92ce4188da92edbca9e7533b3483b73c7)